pages/header_section.py

```diff
@@ -12,7 +12,7 @@ def click_sign_in_button(self):
         self.click(by_locator=self.__HEADER_SIGNIN_BUTTON)
 
     def check_if_nickname_is_presented(self):
-        return self.get_element(by_locator=self.__HEADER_NICKNAME_TEXT)
+        self.if_element_present(by_locator=self.__HEADER_NICKNAME_TEXT)
 
     def click_logout_button(self):
-        self.click(by_locator=self.__HEADER_LOGOUT_BUTTON)
\ No newline at end of file
+        self.click(by_locator=self.__HEADER_LOGOUT_BUTTON)
```

Comment:
to be consistent with declaring return types, functions that don't return anything may be written this way:
```python
    def click_logout_button(self) -> None:
```